src/app/body/task-details/task-details.component.html

```diff
@@ -1,4 +1,4 @@
-<ng-container *ngIf="(taskDataObservable | async) && (activityData | async)" class="task-position">
+<ng-container *ngIf="(taskDataObservable | async)" class="task-position">
```

Comment:
Made this change to prevent entire page from reloading when changing the tabs in the botton